app/views/sources/edit.html.haml

```diff
@@ -0,0 +1,13 @@
+%h1 Edit Source
+
+= form_for source do |f|
+  .form-group
+    = f.label :name
+    = f.text_field :name, { class: 'form-control', required: true }
+
+  - for type in [:email_rank, :location_rank, :organization_name_rank, :phone_number_rank, :website_rank]
+    .form-group
+      = f.label type
+      = f.select(type, source_rank_options(type), {}, { class: 'form-control' })
```

Comment:
Right you are, but you couldn't find it because I made it up! 😃 

https://github.com/artsy/bearden/blob/master/app/helpers/application_helper.rb#L15

See also: c1c4719